README.md

```diff
@@ -105,7 +144,7 @@ directory.
 To install, just copy this one executable into a directory that is
```

Comment:
Right above here, should we tweak this to suggest building release and linking the runtime static like we did on the other page?